### ISSUE TITLE
What's next for NaiveUI ?

### ISSUE THREAD

**Issue by jimmysdr** [tusen-ai/naive-ui#5346]
### This function solves the problem (这个功能解决的问题)
I know NaiveUI is a very good library, and as you know, thousands of projects rely on GetX, and its stability is critical to the community. If you find it difficult to allocate enough time, maybe you can consider creating a GitHub discussion to maintain the package. This way, the community can contribute to its maintenance and not just depend on your availability

We have seen examples in the past where project maintainers have found it beneficial to delegate tasks and responsibilities within the community due to time constraints. This approach not only lightens their workload, but also ensures the continued growth and stability of the project.

Your expertise and leadership on this project has been invaluable, and I believe this shift in strategy will help maintain and enhance the reputation of the package in the developer community.

What I can see is that NiaveUI has accumulated a lot of issues and Pull requests, and I hope the author can involve the community instead of maintaining it by only one person, which is not friendly to the sustainable development of a library. A discussion thread or Issue can be opened to recruit more people to solve and review problems. This will also distract you and keep the library running steadily

**Comment by ChronicStone** [comment]
@jimmysdr Very good point. Naive UI is really excellent and already fairly complete, and benefits from having a large user base already.

But it looks like it's been a moment already since @07akioni has other priorities & isn't able to manage the issues / PRs as much as before, which is totally normal, the project is huge & has been maintained with very few people since quite a while which is a huge burden.

It would be great if we could discuss of steps we can take to shift how the library is maintained to involve much more its community. 

About the current issue count, for sure something must be done. It has reached a point where it looks unmanageable to handle. Don't remember which repo, but a big open-source project has recently wiped entirely its issues (closed everything) to start fresh. Maybe we can do something similar here, but a new flow for managing these issues needs to be in place beforehand.

The part that worries me is the fact that with this growing stack of unhandled issues / unmerged PRs, the project ends up appearing as unmaintained. I've been using Naive UI several jobs until now, and it being well maintained was one of the reasons I could confidently justify my choice to clients. This is to me the best UI library for Vue out there, so loosing it would be a huge loss.

From a personal perspective, I've been using Naive UI extensively for two years now, it's really an awesome library, and I'd like to start giving back what it brought to me by contributing. One thing that hits me, and might be an issue to get the community actively involved for the lib's maintenance is the fact that a very big percentage of issues are written in chinese. I know a big part of Naive UI's community is chinese, that prevents by default the non-chinese part of the community from helping handling issues / questions / PRs ...

Anyway I'll just tag the current biggest contributors to get some attention so this discussion can go forward : 
@07akioni @amadeus711 @Sepush @JiwenBai @Talljack @Fnzero

**Comment by jahnli** [comment]
Naive UI has been widely used, and I recommend it to the company. I think it is the best UI library in Vue. There is no one. The recent large open source project you mentioned that completely eliminates the problem may be lodash. NavieUI needs more. Only when people maintain and contribute can it achieve good development

I'm not sure if the problem that prevents participation in library maintenance is because it is written in Chinese. I remember that the documentation and code include both Chinese and English.

I am also contributing something to NavieUI to the best of my ability to try to make it better, and hope that more capable people will participate.

**Comment by codeDevStory** [comment]
感觉好久都没维护了

**Comment by 07akioni** [comment]
> 感觉好久都没维护了

Yes, I've been busy for quite a long time. I'll put some time here again since I'm not that busy (although not as much as I was a student).

**Comment by codeDevStory** [comment]
> > 感觉好久没有维护了
> 
> 是的，我已经忙了很长一段时间了。我会再花一些时间在这里，因为我没那么忙（尽管没有我学生时那么忙）。

感谢作者

**Comment by 07akioni** [comment]
> @jimmysdr Very good point. Naive UI is really excellent and already fairly complete, and benefits from having a large user base already.
> 
> But it looks like it's been a moment already since @07akioni has other priorities & isn't able to manage the issues / PRs as much as before, which is totally normal, the project is huge & has been maintained with very few people since quite a while which is a huge burden.
> 
> It would be great if we could discuss of steps we can take to shift how the library is maintained to involve much more its community.
> 
> About the current issue count, for sure something must be done. It has reached a point where it looks unmanageable to handle. Don't remember which repo, but a big open-source project has recently wiped entirely its issues (closed everything) to start fresh. Maybe we can do something similar here, but a new flow for managing these issues needs to be in place beforehand.
> 
> The part that worries me is the fact that with this growing stack of unhandled issues / unmerged PRs, the project ends up appearing as unmaintained. I've been using Naive UI several jobs until now, and it being well maintained was one of the reasons I could confidently justify my choice to clients. This is to me the best UI library for Vue out there, so loosing it would be a huge loss.
> 
> From a personal perspective, I've been using Naive UI extensively for two years now, it's really an awesome library, and I'd like to start giving back what it brought to me by contributing. One thing that hits me, and might be an issue to get the community actively involved for the lib's maintenance is the fact that a very big percentage of issues are written in chinese. I know a big part of Naive UI's community is chinese, that prevents by default the non-chinese part of the community from helping handling issues / questions / PRs ...
> 
> Anyway I'll just tag the current biggest contributors to get some attention so this discussion can go forward : @07akioni @amadeus711 @Sepush @JiwenBai @Talljack @Fnzero

Many thanks. I'll handle all these things. I may close some issues that won't be implemented.

However the main concern is for thing like UI library, there must be a main maintainer. I've been working as internship at ant-design before. Even for the biggest react ui library in China, maintaining work is mainly done by few full-time employees.

I won't making any money by maintaining the library (I've much want to do than just an UI library). For me it's just about interest, and a place to validate most advanced ideas for to-business UI coding (such a heavy UI framework is not the best choice to customers).

I am open if some one want to be a co-maintainer of the library. If any one is interested in this & has passion & ability to do these. You can reach out me personally at discord or any place.

I've no idea how naive-ui will go. The things can be confirmed are these:

1. I'll still maintain the library for a very long time.
2. Issues & PR will be eventually resolved (although not a high speed).
3. Large features are not planned (since my time is somehow limited)
4. Versioning will be more frequently than 2023 in 2024.

Thanks for all your supporting!
